jira/resources.py

```diff
@@ -755,7 +755,7 @@ def delete(self, moveFixIssuesTo=None, moveAffectedIssuesTo=None):
         if moveAffectedIssuesTo is not None:
             params['moveAffectedIssuesTo'] = moveAffectedIssuesTo
 
-        super(Version, self).delete(params)
+        return super(Version, self).delete(params)
```

Comment:
Well, no return value was documented and no test were using it, so this cannot count as an API change.